README.md

```diff
@@ -1 +1,102 @@
-# Apache Ignite 3.x
+Apache Ignite 3.x
+===
+
+Apache Ignite 3 is a distributed database for high-performance computing with in-memory speed.
+
+* MULTI-TIER STORAGE: Ignite scales up and out across memory and disk. By default, Ignite operates in a pure in-memory mode. But, by toggling a single configuration setting, you can turn a cluster into a database that can grow beyond the cluster's memory capacity.
+* DISTRIBUTED SQL: Use Ignite as a traditional SQL database by leveraging JDBC drivers, ODBC drivers, or the native SQL APIs that are available for Java, C#, C++, Python, and other programming languages.
+* ACID TRANSACTIONS: Ignite can operate in a strongly consistent mode that provides full support for distributed ACID transactions.
+* COMPUTE APIS: With traditional databases, for in-place calculations, you use stored procedures that are written in a language such as PL/SQL. With Ignite, you use modern JVM languages, C# or C++ to develop and execute custom tasks across your distributed database.
+* MACHINE LEARNING: Ignite machine learning uses built-in algorithms and tools, as well as TensorFlow integration, to enable the building of scalable machine learning models and avoid costly data transfers.
```

Comment:
Just a copy&past from Apache Ignite 2 There is no any ML support in Apache Ignite 3 yet.